en/V5-Hardware_Platform_Requirements.md

```diff
@@ -2,9 +2,9 @@
 
 ## Control Objective
```

Comment:
Small comment: this chapter provides the requirements for hardware platform such that a secure hardware configuration can be guaranteed. For example,

3.1.4 discusses correctly configuring secure boot. 5.1.2 requires that the platform supports this.
5.1.1 requires that the platform supports disabling debug interface. 1.2.4 requires that it is done so in production.
Can you take up this nuance in the description?
Thanks!

Reply:
I'm adding this, but we don't have the same kind of summary in other sections.